File: Project Euler/Multiples of 3 or 5/pedro_chagas_5485.c
#include <stdio.h>

int solve(int limit) {

  int multiples_sum = 0;
  for (int i = 0; i < limit; i++){
    if (i % 3 ==0 || i % 5 == 0){
      multiples_sum++;
      }
  }
  return multiples_sum;
}

int main(){

  int solution = solve(1000);
  printf("%d\n", solution);
  return 0;
}

File: Project Euler/Multiples of 3 or 5/pedro_chagas_5485.py
"""Resolução do primeiro problema do site projecteuler.net"""


def solve(limit: int) -> int:
    """
    Função principal para resolver o problema, recebe o limite
    como input e entrega a quantidade de múltiplos de 3 ou 5.
    """

    multiples_sum = 0
    for i in range(limit):
        if i % 3 == 0 or i % 5 == 0:
            multiples_sum += 1
    return multiples_sum


if __name__ == "__main__":
    solution = solve(1000)
    print(solution)
